docker-compose/README.md

```
@@ -0,0 +1,266 @@
+# Using Python
+python -c "import base64; import os; print(base64.urlsafe_b64encode(os.urandom(32)).decode())"
+
+# Using OpenSSL
+openssl rand -base64 32
+```
+
+## 🌐 Access Your Services
+
+After running the Docker Compose command:
+
+- **Exosphere Dashboard**: http://localhost:3000
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)